Makefile

```diff
@@ -69,16 +69,16 @@ clean:
 libp2p_helper:
```

Comment:
there's a target `codaslim` in the Makefile that I think you can delete. There's also a reference to it in the `PHONY` section.

Reply:
Also in the `PHONY` section there are mentions of other `coda*` targets that no longer exist, those can be deleted

Reply:
I think also `codabuilder` can be renamed to `minabuilder` (although I think the in-docker builds are broken right now)